CHANGELOG.md

```diff
@@ -1,13 +1,28 @@
 ## CHANGELOG
 
+### 2.x.x (In-progress)
+
+**SDK**
+
+- **Feature** Updated `getUserName` and `getProfilePhoto` interfaces to be asynchronous. Old methods are deprecated
+- **Feature** Updated `MiniApp().shared.create` interface to accept another optional query string parameter.
```

Comment:
Seems that these are from this PR? https://github.com/rakutentech/ios-miniapp/pull/199/files

Reply:
yes, as the 2 PR were going to be validated at the same time, I knew there would be a conflict so I cherry picked the changelog to avoid it.